src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4824,6 +4825,7 @@ .ForwardHelpCategory Cmdlet
         internal const ActionPreference defaultVerbosePreference = ActionPreference.SilentlyContinue;
         internal const ActionPreference defaultWarningPreference = ActionPreference.Continue;
         internal const ActionPreference defaultInformationPreference = ActionPreference.SilentlyContinue;
+        internal const Microsoft.PowerShell.FileEncoding defaultFileEncodingPreference = FileEncoding.Unknown;
```

Comment:
I can find no references to this field other than setting the session state variable. the variable. How is it used and how does it control behavior? Currently, it appears to be a NOP.